cargo-dist/src/init.rs

```diff
@@ -234,24 +289,6 @@ pub fn do_init(cfg: &Config, args: &InitArgs) -> DistResult<()> {
     let root_workspace = workspaces.root_workspace();
     let check = console::style("✔".to_string()).for_stderr().green();
 
-    eprintln!("let's setup your dist config...");
-    eprintln!();
-
-    // For each [workspace] Cargo.toml in the workspaces, initialize [profile]
-    let mut did_add_profile = false;
-    for workspace_idx in workspaces.all_workspace_indices() {
-        let workspace = workspaces.workspace(workspace_idx);
-        if workspace.kind == WorkspaceKind::Rust {
-            let mut workspace_toml = config::load_toml(&workspace.manifest_path)?;
-            did_add_profile |= init_dist_profile(cfg, &mut workspace_toml)?;
-            config::write_toml(&workspace.manifest_path, workspace_toml)?;
-        }
-    }
-
-    if did_add_profile {
-        eprintln!("{check} added [profile.dist] to your workspace Cargo.toml");
-    }
-
```

Comment:
This is moved later in the function, to avoid `dist init` being canceled but still overwriting `Cargo.toml`.